GitUI/BranchTreePanel/RepoObjectsTree.ContextActions.cs

```diff
@@ -59,8 +59,8 @@ private void EnableMoveTreeUpDownContexMenu(bool hasSingleSelection, NodeBase se
             bool isSingleTreeSelected = hasSingleSelection && selectedNode is Tree;
             TreeNode treeNode = (selectedNode as Tree)?.TreeViewNode;
```

Comment:
This should be marked as nullable as well
```suggestion
            TreeNode? treeNode = (selectedNode as Tree)?.TreeViewNode;
```